games-emulation/zsnes/zsnes-2.0.12.ebuild

```diff
@@ -0,0 +1,76 @@
+# Copyright 2021-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit desktop flag-o-matic multilib toolchain-funcs xdg-utils
```

Comment:
If you use `xdg` instead of `xdg-utils` then you don't need to add the phase functions below.

Reply:
Thanks! I've switched it to `xdg` in 9653e39.